concrete/src/Notification/Alert/Filter/FilterList.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace Concrete\Core\Notification\Alert\Filter;
+
+class FilterList
+{
+
+    /**
+     * @var FilterInterface[]
+     */
+    protected $filters = [];
+
+    /**
+     * @param FilterInterface $filter
+     */
+    public function addFilter(FilterInterface $filter)
+    {
+        $this->filters[] = $filter;
+    }
+
+    /**
+     * @return FilterInterface[]
+     */
+    public function getFilters()
+    {
+        return $this->filters;
+    }
+
+    /**
+     * @return void
+     */
+    public function orderEntries()
+    {
+        usort(
+            $this->filters,
+            function ($a, $b) {
+                return strcasecmp($a->getName(), $b->getName());
+            }
+        );
+    }
+
+    /**
+     * @param string $key
+     * @return FilterInterface
```

Comment:
The return type for this should be `FilterInterface|null`

Reply:
👍